_targets.R

```diff
@@ -0,0 +1,64 @@
+# Created by use_targets().
+# Follow the comments below to fill in this target script.
+# Then follow the manual to check and run the pipeline:
+#   https://books.ropensci.org/targets/walkthrough.html#inspect-the-pipeline
+
+# Load packages required to define the pipeline:
+library(targets)
+
+# Set target options:
+tar_option_set(
+  packages = c("tibble") # Packages that your targets need for their tasks.
+  # format = "qs", # Optionally set the default storage format. qs is fast.
+)
+
+# Run the R scripts in the R/ folder with your custom functions:
+tar_source()
+source(here::here("data-raw/algorithm.R"))
+source(here::here("data-raw/variable-description.R"))
+
+# Replace the target list below with your own:
+list(
+  tar_target(
+    name = algorithm_csv,
+    command = "data-raw/algorithm.csv",
+    format = "file"
+  ),
+  tar_target(
+    name = algorithm,
```

Comment:
```suggestion
    name = algorithm_data,
```
To align naming with the function `read_algorithm_data` below

Reply:
The name here would be the name that we use internally to access those objects (e.g. with using `use_data()`. It is a bit redundant to add `_data` to the end of the object when it is a data frame already (it would be a bit like calling all variables that are strings with `_string` like `age_string` or `drugs_string`).